### PR TITLE
README.md: 

### PR DESCRIPTION
**Pull request by LetschM**
I finished readme.md but I have the feeling I forgot something anywhere in between (cannot say what exactly). Can you please look through it and give feedback whether it is complete or not?

### CODE REVIEW

**Comment by LetschM** [comment]
What again do I have to do with git submodule init and git submodule update?

**Comment by zuphilip** [comment]
Hm... the squash did not work as expected. Let us look at this togeter later these days...

**Comment by zuphilip** [comment]
@LetschM Did you change anything else except the README.md in this Pull Reques? Maybe it is easier to start with the clean actual state and add your latest README.md manually, commit and push again (force or in another branch) and create a new pull request. We can make this together if you prefer.

**Comment by LetschM** [comment]
No I didn't but in this one
https://github.com/LetschM/infolis-web/commit/cff1ef654e55b0708493d191a8b13d51a5b7cf0f
 there are some more changes aside from README.md which I assume were made
automatically? If these changes aren't important we can start again with a
clean state.

2015-12-08 16:56 GMT+01:00 Philipp Zumstein notifications@github.com:

File: README.md [hunk 1]
```diff
@@ -15,3 +15,120 @@ These are the licenses of the bundled libraries, for those pulled in using
 +---------------------------------------------+--------------|
```
Comment (kba) [inline README.md]: The table can be removed as well.

**Pull request merged**